### ISSUE TITLE
Reverse countdown option

### ISSUE THREAD

**Issue by MrBertie** [tberghuis/FloatingCountdownTimer#9]
Love the simplicity and usefulness of this app, thank you.

Would you consider adding the option to reverse the countdown direction, starting from 0 and counting _up_ to the total, instead of counting down only (which also has a valuable use case)?

I feel that it would be useful if you are timing a talk, lecture, or lesson, and need to keep track of where you are, e.g. via timing marks in your script.

**Comment by tberghuis** [comment]
You can start both a stopwatch and countdown timer at the same time.

**Comment by MrBertie** [comment]
That is true but having 2 counters running in different directions on your display might be confusing, after all the man with 2 watches never really knows what time it is...

**Comment by Clakie** [comment]
Even if only had one on screen I can see the use of needing a countdown timer but being able to quickly at a glance with no mental energy required know how long a timer has been running.

Say I'm timing something but I have a total maximum time I have to do it and I want that alert but while that timer is going I have to do certain things based on variable conditions in relation to how long it's been since this task has started. I also need to keep track of how long something else has been going (this would use the stop watch)

Could also see this as a use for more than one of each type of timer (you could put this behind a pay wall as I imagine you'd want to use different colours for each timer) have an add timer button each with their own colour, time value, timer type, timer direction. If you add lots of customisation you could hide the options behind a 🔽🔼 style drop down menu to reduce gui clutter
